CI/build-windows.ps1

```diff
@@ -0,0 +1,142 @@
+Param(
+    [Switch]$Help,
+    [Switch]$Quiet,
+    [Switch]$Verbose,
+    [Switch]$NoChoco,
+    [Switch]$Package,
+    [Switch]$SkipDependencyChecks,
+    [Switch]$BuildInstaller,
+    [Switch]$CombinedArchs,
+    [String]$BuildDirectory = "build",
+    [String]$BuildArch = (Get-CimInstance CIM_OperatingSystem).OSArchitecture,
+    [String]$BuildConfiguration = "RelWithDebInfo"
+)
+
+##############################################################################
+# Windows OBS build script
+##############################################################################
+#
+# This script contains all steps necessary to:
+#
+#   * Build OBS with all required dependencies
+#   * Create 64-bit and 32-bit variants
+#
+# Parameters:
+#   -Help                   : Print usage help
+#   -Quiet                  : Supress most build process output
+#   -Verbose                : Enable more verbose build process output
+#   -SkipDependencyChecks   : Skips dependency checks
```

Comment:
"Skip dependency checks" as in other files.

Reply:
Fixed.